lib/src/presentation/view/card_list.dart

```diff
@@ -0,0 +1,341 @@
+import 'package:flutter/material.dart';
+import '../../core/util/card_constants.dart';
+import '../../data/model/card.dart';
+import '../bloc/card_bloc.dart';
+import 'card_page.dart';
+
+class CardList extends StatefulWidget {
+  final CardBloc cardBloc;
+
+  const CardList({
+    required this.cardBloc,
+    super.key,
+  });
+
+  @override
+  State<CardList> createState() => _CardListState();
+}
+
+class _CardListState extends State<CardList> {
+  final _totemUrl = kRacesTotemEndpointApi;
+  final _demonUrl = kRacesDemonEndPointApi;
+  final _murlocUrl = kRacesMurlocEndPointApi;
```

Comment:
If you define this three variables as consts, why do you create a new variable to assign this constants. Instead to do this you can use directly the const that you already define.

Reply:
You are right. Done